src/features/island/collectibles/components/Wardrobe.tsx

```diff
@@ -4,45 +4,34 @@ import { PIXEL_SCALE } from "features/game/lib/constants";
 import { SUNNYSIDE } from "assets/sunnyside";
 import { Modal } from "components/ui/Modal";
 import { CloseButtonPanel } from "features/game/components/CloseablePanel";
-import { StylistWearables } from "features/world/ui/stylist/StylistWearables";
-import { BASIC_WEARABLES } from "features/game/types/stylist";
-import { OuterPanel } from "components/ui/Panel";
+import { Button } from "components/ui/Button";
+import { useAppTranslation } from "lib/i18n/useAppTranslations";
+import { Label } from "components/ui/Label";
+import lockIcon from "assets/icons/lock.png";
 
-function hasOpened() {
-  return !!localStorage.getItem("hasOpenedWardrobe");
```

Comment:
Recommend:
Should probably have a list in the front-end of obsolete `localStorage` keys and attempt to remove associated data from `localStorage` for each of those keys.

Strawman:
1. create map of name => date, where date is an "obsolete start date"
2. add a function that attempts to remove all keys in the map
3. call the function on home island load
4. in mid 2025, start removing keys with an "obsolete start date" older than 6 months